background.ts

```diff
@@ -1397,6 +1397,10 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
     doFactoryReset(keepSystemImages);
   }
 
+  async k8sReset(context: CommandWorkerInterface.CommandContext, mode: 'fast' | 'wipe') {
+    return await doK8sReset(mode, context);
```

Comment:
🍰 Reading that, we may end up skipping if the backend is busy (i.e. in the middle of restarting). That should produce a 412 or 422?